components/JobsSidebarCTA.jsx

```diff
@@ -12,17 +12,19 @@ export const JobsSidebarCTA = ({ isUser }) => {
             background="linear-gradient(125.04deg, #1D407C -0.1%, #4B1114 97.61%)"
             borderRadius="20px"
           >
-            <Heading marginBottom={3} size="md">
-              Want to post a job?
-            </Heading>
-            <Text marginBottom={6}>
-              All you need to do is sign up! It’s free and easy, just make sure
-              you check our{" "}
-              <Link as={NextLink} href={"/rules-and-faq"} passHref>
-                Rules and FAQ
-              </Link>{" "}
-              before posting.
-            </Text>
+            <Box color={"white"}>
```

Comment:
Hey @JustinReidy nice find! I'm wondering though, did you try adding the color={"white"} to the containing parent element above it? The Box element that should start on line 10? I would suspect that to work as well, but I haven't gone tinkering myself.

Reply:
I did try adding it to the parent element. However when I do, the button linking to "Sign Up" turns white. Might be a better way of resolving this. Adding a box around it was just the easy way to work around it.
![image](https://user-images.githubusercontent.com/16121783/193513919-96ebdcd7-d6f4-40ef-8539-5c8ecd2d1769.png)

Reply:
@JustinReidy ah, of course. Hmm, how about tossing `color={"white"}` on the Heading and Text elements that we would like to always be white?